appstore/model.go

```diff
@@ -48,14 +57,23 @@ type (
 		ProductID                 string `json:"product_id"`
 		TransactionID             string `json:"transaction_id"`
 		OriginalTransactionID     string `json:"original_transaction_id"`
-		IsTrialPeriod             string `json:"is_trial_period"`
 		AppItemID                 string `json:"app_item_id"`
 		VersionExternalIdentifier string `json:"version_external_identifier"`
 		WebOrderLineItemID        string `json:"web_order_line_item_id"`
+
+		IsTrialPeriod                  string `json:"is_trial_period"`
+		SubscriptionExpirationIntent   string `json:"expiration_intent"`
```

Comment:
These fields are not shown in ``InApp`` field, but shown in ``PendingRenewalInfo`` field.
Is it necessary to add them to ``InApp``?

The following is the raw response from AppStore.
```
  "pending_renewal_info": [
    {
      "auto_renew_product_id": "my_product_id",
      "auto_renew_status": "0",
      "expiration_intent": "1",
      "is_in_billing_retry_period": "0",
      "product_id": "my_product_id"
    }
  ],
  "receipt": {
    "adam_id": 11111111111,
    "app_item_id": 11111111111,
    "application_version": "100",
    "bundle_id": "my_bundle_id",
    "download_id": 11111111111,
    "in_app": [
      {
        "expires_date": "2017-07-21 15:00:39 Etc/GMT",
        "expires_date_ms": "1500649239000",
        "expires_date_pst": "2017-07-21 08:00:39 America/Los_Angeles",
        "is_trial_period": "false",
        "original_purchase_date": "2017-05-21 15:00:39 Etc/GMT",
        "original_purchase_date_ms": "1495378839000",
        "original_purchase_date_pst": "2017-05-21 08:00:39 America/Los_Angeles",
        "original_transaction_id": "11111111111",
        "product_id": "my_product_id",
        "purchase_date": "2017-06-21 15:00:39 Etc/GMT",
        "purchase_date_ms": "1498057239000",
        "purchase_date_pst": "2017-06-21 08:00:39 America/Los_Angeles",
        "quantity": "1",
        "transaction_id": "11111111111",
        "web_order_line_item_id": "11111111111"
      },
...
```